SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -173,16 +173,29 @@ void USpatialStatics::PrintTextSpatial(UObject* WorldContextObject, const FText
 	PrintStringSpatial(WorldContextObject, InText.ToString(), bPrintToScreen, TextColor, Duration);
 }
 
-FString USpatialStatics::GetActorEntityIDString(const AActor* Actor)
+int64 USpatialStatics::GetActorEntityId(const AActor* Actor)
 {
 	if (Actor != nullptr)
 	{
 		if (const USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(Actor->GetNetDriver()))
```

Comment:
I'd flip this around and return early if it's nullptr if we want this to be valid. If we don't have a valid reason for calling this when we don't have a net driver should we also just make this a check?

Reply:
Check on actor and net driver then, good catch.